### PR TITLE
Unlink account resource

### PR DESCRIPTION
**Pull request by nmonterroso**
@Wikia/services-team 
https://wikia-inc.atlassian.net/browse/SERVICES-979

Add a resource to unlink a user's Google account from their Wikia account

### CODE REVIEW

File: service/external-auth/src/main/java/com/wikia/services/externalauth/service/GoogleAuthService.java [hunk 1]
```diff
@@ -70,4 +70,12 @@ public boolean connectUsers(String userId, String accessToken) {
     LinkedGoogleAccount linkedGoogleAccount = new LinkedGoogleAccount(userId, account.get().getUserId());
     return storedGoogleAccounts.add(linkedGoogleAccount);
   }
+
+  public boolean disconnectAccount(String userId, String accessToken) {
+    AuthenticatedAccount googleAccount = googleGateway.validateToken(accessToken)
```
Comment (ArturKlajnerok) [inline service/external-auth/src/main/java/com/wikia/services/externalauth/service/GoogleAuthService.java]: Should we require to have google access token for account unlinking ? My understanding is that if one of our clients have valid wikia access token we shouldn't force it to provide the google one for this action.

**Comment by ArturKlajnerok** [comment]
Other than one small comment, Ltgm :+1:

File: service/external-auth/src/main/java/com/wikia/services/externalauth/service/GoogleAuthService.java (hunk 1 above)
Comment (nmonterroso) [inline service/external-auth/src/main/java/com/wikia/services/externalauth/service/GoogleAuthService.java]: Yah, good point. I'll change it.

File: service/external-auth/src/main/java/com/wikia/services/externalauth/resources/GoogleAuthResource.java [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import com.wikia.pandora.core.swagger.SwaggerBundleProvider;
 import com.wikia.pandora.http.StandardResponses;
 import com.wikia.services.externalauth.config.TokenCookieConfig;
+import com.wikia.services.externalauth.dao.StoredGoogleAccounts;
```
Comment (WikiaTech) [inline service/external-auth/src/main/java/com/wikia/services/externalauth/resources/GoogleAuthResource.java]: MINOR Remove this unused import 'com.wikia.services.externalauth.dao.StoredGoogleAccounts'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by ArturKlajnerok** [comment]
Lgtm :+1:

**Pull request merged**